data/ChinatuL.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "Chinatu Lucia Eke",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Web Developer | Open Source Enthusiast",
+  "links": [
+    {
+      "name": "Twitter",
+      "url": "https://twitter.com/ChinatuLucia",
+      "icon": "FaTwitter"
+    },
+    {
+      "name": "Instagram",
+      "url": "https://www.instagram.com/chinatulucia/",
+      "icon": "FaInstagram"
+    },
+    {
+      "name": "LinkedIn",
+      "url": "https://www.linkedin.com/in/chinatu-lucia-3b923b1b1/",
+      "icon": "FaLinkedIn"
```

Comment:
Wrong icon name.
```suggestion
      "icon": "FaLinkedin"
```

Reply:
Thank you very much for your review. I would effect that immediately